Decisions.TruCap/Steps/AuthSteps.cs

```diff
@@ -38,44 +39,75 @@ public class AuthSteps
 
             try
             {
-                HttpResponseMessage response = await client.SendAsync(request);
+                HttpResponseMessage response = client.Send(request);
                 response.EnsureSuccessStatusCode();
 
-                return LoginResponse.JsonDeserialize(await response.Content.ReadAsStringAsync());
+                Task<string> resultTask = response.Content.ReadAsStringAsync();
+                resultTask.Wait();
+                return LoginResponse.JsonDeserialize(resultTask.Result);
             }
-            catch (Exception ex)
+            catch (BusinessRuleException ex)
```

Comment:
This won't ever work because the code above will never throw a `BusinessRuleException`